File: docs/examples/quickstart/java/allow-01.java
import com.osohq.oso.Oso;

class Allow {
    public static void main(String[] args) {
        Oso oso = new Oso();
        String actor = "alice@example.com";
        Expense resource = Expenses.EXPENSES[1];
        boolean allowed = oso.isAllowed(actor, "GET", resource);
    }

}

File: docs/examples/quickstart/python/allow-01.py
from oso import Oso
from expense import EXPENSES


def setup_oso():
    oso = Oso()
    return oso


oso = setup_oso()
oso = Oso()
actor = "alice@example.com"
resource = EXPENSES[1]
oso.is_allowed(actor, "GET", resource)
